docs/app/Component/ComponentProps.js

```diff
@@ -29,42 +43,28 @@ export default class ComponentProps extends Component {
       }
       type = type && `{${type}}`;
 
-      let required = propConfig.required && (
-          <span className='ui mini red circular label'>required</span>
-        );
-
-      // default value
-      if (propConfig.defaultValue) {
-        defaultValue = (
-          <span>{propConfig.defaultValue.value}</span>
-        );
-        defaultIsComputed = propConfig.defaultValue.computed && (
-            <span className='ui mini gray circular label'>computed</span>
-          );
-      }
+      let required = propConfig.required;
+      let defaultValue = propConfig.defaultValue;
 
-      return (
-        <div key={propName}>
-          <div style={{display: 'flex'}}>
-            <div style={{flex: '4'}}>{name} {required}</div>
-            <div style={{flex: '2'}}>{type}</div>
-            <div style={{flex: '4'}}>{defaultValue} {defaultIsComputed}</div>
-            <div style={{flex: '8'}}>{description}</div>
-          </div>
-        </div>
-      );
+      return {
+        name,
+        type,
+        value,
+        required,
+        defaultValue,
+        description,
+      };
     });
 
     return (
-      <Segment className='basic vertical'>
+      <Segment className='vertical'>
         <div className='ui header'>Props</div>
-        <div style={{display: 'flex'}}>
-          <b style={{flex: '4'}}>name</b>
-          <b style={{flex: '2'}}>type</b>
-          <b style={{flex: '4'}}>default</b>
-          <b style={{flex: '8'}}>description</b>
-        </div>
-        {content}
+        <Table data={content} className='very basic'>
+          <TableColumn dataKey='name' cellRenderer={this.nameRenderer} />
+          <TableColumn dataKey='type' />
+          <TableColumn dataKey='defaultValue' cellRenderer={this.defaultValueRenderer} />
+          <TableColumn dataKey='description' />
+        </Table>
```

Comment:
Here is the new PropTypes table, each column uses some key from each object in the array of PropTypes.  Columns with renderer functions (defined above) are combining multiple properties (like `name` and `required`) into a single column.